cstring.h

```diff
@@ -243,6 +247,13 @@ char stringCharToUpper(char c) {
     return c;
 }
 
+int stringCharToInt(char c) {
+    if ('0' <= c && c <= '9') 
+        return c - '0';
+    setError(ERR_INVALID_NUMBER_REPR);
```

Comment:
If function can change error state, it have to clear error in the begin, to prevent smth like that:
```
// err = no_err
foo()                              // set err to smth, but result wasn't checked
...
//err = some_err
bar()                              // no error actually happened but flag wasn't cleaned
if (err == some_err) {             // check error in bar() result
    ...                            // but goes here because err after foo()
}
```

Reply:
although it is better to always check the error after each function call